documentation.md

```diff
@@ -1,157 +1,110 @@
+```yaml
+- name: Early Career Researchers
+  files:
+    - project-design/project-design
+    - collaboration/github-novice
+    - project-design/project-repo
+...
+  colour: blue
+  description: Early Career Researchers (ECRs) are students, PhDs and early-stage
+    postdocs who may not have a lot of experience in contributing to research
+    projects from start to finish. This curated set of chapters will allow them
+    to understand what best practices apply at different stages of development
+    to ensure research reproducibility.
+```
 
->**Important:** Also change the string in the `bio-turingway/pathways/tests/test_card.py` to make sure the test passes.  
->
-> Changing this will require reinstallation of the dependencies, see "Part 03 - Dependencies and Library Install" below for futher details. 
+The pathways file is a list of pathways.
+Each pathway has the following keys,
 
-### How many pages linked in each card
+name
+: The pathway name.
+: This will be used in the tags, the landing page title, and the card.
 
-Currently the first four pages for each profile/pathway are linked within the card. Pages after this are not included. 
+files
+: Relative paths of the pages to include (excluding `.md` suffix).
+: Note that if you include a page without its parents it will not appear in the landing page.
+  So it is not possible to have `communications/comms-overview/comms-overview-principles` without `communications/comms-overview/`.
+  Pages missing from the landing page will still be tagged with badges.
 
-To alter the number of included pages in the card, look for a function within `card.py` called `create_bullet_string` and change the number in the `for` (Line 37) and `if` (Line 40) from 3 to another value. 
+colour
+: The colour of the pathway's badge.
+: All CSS colour names should be supported.
 
-<img src="images/image_cardfunction.png" alt="drawing" width="800"/>
+description
+: Text describing the profile which the pathway was designed for.
+: This text is displayed on the pathway's landing page.
 
+## The Card Panel on the Welcome Page
 
-> Changing this will require reinstallation of the dependencies, see "Part 03 - Dependencies and Library Install" below for futher details. 
+The pathways script adds new (HTML) pages to the JupyterBook build.
+A landing page for each of the pathways defined in `profiles.yml` is created.
+A panel of pathways cards is also produced and added to the Welcome Page.
 
-### Edit the appearance of the cards and the panel
+### Changing Where the Cards Appear
 
-Within `card.py`, edits can be made to change the appearance of the cards and panels. You can find the guidelines for this in the following documentation:
+Currently, the cards are created under the heading Different Pathways.
+The script `pathways/pathways/card.py` looks for the line of text `## Different Pathways` in the Welcome Page markdown file and inserts the card panels after it.
```

Comment:
In future, we may want to change this to look for something like:

```
<!--- pathways start --->

<!--- pathways stop --->
```

I'm thinking similar to how the all-contributors bot injects the table into the README of the main repo.

Not critical for this PR, just a thought for the future :)

Reply:
Great idea.

There is a fair bit of hardcoded values/expectations so there is room to make things more general/reusable.
I would quite like to have something more like directives to place pathways elements around the book. Might be able to avoid a two step build that way.

I've opened an [issue](https://github.com/the-turing-way/pathways/issues/5) 🙏.